reenact/static/css/project.css

```diff
@@ -11,3 +9,59 @@
   background-color: #f2dede;
   border-color: #eed3d7;
 }
+
+.nav-tab ul {
+  list-style: none;
+  padding-left: 0;
+  z-index: 1000;
+}
+
+.nav-tab ul li {
+  display: inline-block;
+  text-align: left;
+}
+
+.nav-tab a {
+  display: block;
+  text-decoration: none;
+}
+
+.nav-tab ul ul {
+  display: none;
+  position: absolute;
+  padding: 8px 0 8px 0;
+}
+.nav-tab ul ul li {
+  display: list-item;
+  position: relative;
+}
+
+.nav-tab ul li:focus > ul,
+.nav-tab ul li:active > ul {
+  display: inherit;
+}
+
+.nav-tab a:focus + ul,
+.nav-tab a:active + ul {
+  display: inherit;
+}
+
+h1 {
+  font-weight: bold;
+  font-size: 1.5em;
+  display: block;
+}
+
+h2 {
+  font-weight: bold;
+  font-size: 1em;
+  display: block;
+}
+
+/* stupid linter does not allow inline CSS, enjoy confusing styles all over the place */
```

Comment:
I don't think this is stupid as IMO CSS and HTML should be seperated. 
(There are indeed some linter rules which are annoying, but overall I think they improve code readability and maintainability)
In order to not have styles all over the place, we should move those two into a separate CSS file(s).